scandium-core/src/main/java/org/eclipse/californium/scandium/dtls/CertificateRequest.java

```diff
@@ -544,34 +541,6 @@ SignatureAndHashAlgorithm getSupportedSignatureAlgorithm(PublicKey key) {
 		return null;
 	}
 
-	/**
-	 * Truncates a given certificate chain at the first certificate encountered having
-	 * a subject listed in <em>certificateAuthorities</em>.
-	 * 
-	 * @param chain The original certificate chain.
-	 * @return A (potentially) truncated copy of the original chain.
-	 * @throws NullPointerException if the given chain is {@code null}.
-	 */
-	public List<X509Certificate> removeTrustedCertificates(List<X509Certificate> chain) {
```

Comment:
[API break] should we just deprecated it ?

Reply:
FMPOV, that's internal API and not considered to be not adapted.